pkg/config/datafileprojectconfig/entities/entities.go

```diff
@@ -42,7 +42,7 @@ type Experiment struct {
 	TrafficAllocation  []trafficAllocation `json:"trafficAllocation"`
 	AudienceIds        []string            `json:"audienceIds"`
 	ForcedVariations   map[string]string   `json:"forcedVariations"`
-	AudienceConditions interface{}         `json:"audienceConditions"`
```

Comment:
@mikeng13 this change is causing issues for datafiles with audience conditions defined as:
`"audienceConditions": "20413101795"`